### ISSUE TITLE
react/no-unescaped-entities rule is ignored

### ISSUE THREAD

**Issue by MasterCarl** [yannickcr/eslint-plugin-react#2811]
It seems like the `react/no-unescaped-entities` rule isn't being picked up by eslint. I have the following in my `eslintrc.js`:

```
rules: {
	"react/no-unescaped-entities": ["warning", { forbid: [">", "}"] }],
},
```
It seems to be correct as VS Code doesn't show the related warning anymore. However, I still get the following error:

> Failed to compile.

>./src/router.tsx
>  Line 79:22:  `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`  react/no-unescaped-entities

**Comment by ljharb** [comment]
If the CLI is working, and vscode is not, and restarting vscode doesn't fix it, it's a vscode bug.
